apps/www/pages/support.tsx

```diff
@@ -34,181 +62,151 @@ const Index = ({}: Props) => {
+                        clipRule="evenodd"
+                      />
+                    </svg>
+                  }
+                  asChild
                 >
-                  <div className="mb-4">
-                    <h5 className="text-scale-1200 text-lg font-medium">Email Support</h5>
-                    <p className="my-2 block">
-                      <p className="text-scale-1100">
-                        We offer email based support. If you need SLAs, guaranteed response times,
-                        or have an issue, please contact us here.
-                      </p>
-                      <p className="text-scale-1100 mt-2 text-sm">
-                        <span className="text-brand text-sm font-bold">Use this for:</span> Issues
-                        or questions specific to you
-                      </p>
-                    </p>
-                  </div>
+                  <a target="_blank">Join Discord</a>
+                </Button>
+              </Link>
+            </div>
+          </InteractiveShimmerCard>
+        </SectionContainer>
+        <SectionContainer className="!pt-0 max-w-7xl">
+          <div className="mx-auto bg-alternative border rounded-xl p-6 lg:p-10">
+            <div className="grid grid-cols-1 lg:grid-cols-12 gap-2 xl:gap-16 justify-between">
+              <div className="col-span-full lg:col-span-6 gap-2 flex flex-col">
+                <h2 className="text-xl lg:text-2xl tracking-tight">
+                  Can't find what you're looking for?
+                </h2>
+              </div>
+              <div className="col-span-full lg:col-start-7 lg:col-span-6 flex flex-col gap-4">
+                <div className="flex flex-col">
+                  <p className="text-light">The Supabase Support Team is ready to help.</p>
+                  <p className="text-lighter text-sm">
+                    Response time might vary depending on plan type and severity of the issue.
```

Comment:
@awalias @KevinBrolly the purpose of this text is to discourage users on free plans to open too many tickets.
Wdyt? Would you phrase it differently or do you guys think it conveys that message well enough?

Open to suggestions ✌️